backend/webapp/src/main/resources/logback.xml

```diff
@@ -1,15 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
-    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss} [%thread] %-5level %logger{26} - %msg%n%rEx</pattern>
-        </encoder>
-    </appender>
+    <if condition='isDefined("catalina.base")'>
+        <then>
+            <appender name="LOGFILE" class="ch.qos.logback.core.FileAppender">
+                <file>${catalina.base}/logs/ui-designer.log</file>
+                <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+                    <fileNamePattern>ui-designer.%d{yyyy-MM-dd}.log.zip</fileNamePattern>
+                </rollingPolicy>
+                <encoder>
+                    <pattern>%date{HH:mm:ss} [%thread] %-5level %logger{16} - %msg%n%rEx</pattern>
+                </encoder>
+            </appender>
+
+            <root level="INFO">
+                <appender-ref ref="LOGFILE"/>
+            </root>
+        </then>
+        <else>
+            <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder>
+                    <pattern>%date{HH:mm:ss} [%thread] %-5level %logger{16} - %msg%n%rEx</pattern>
+                </encoder>
+            </appender>
+
+            <root level="INFO">
+                <appender-ref ref="CONSOLE"/>
+            </root>
+        </else>
+    </if>
 
-    <root level="INFO">
-        <appender-ref ref="CONSOLE" />
-    </root>
```

Comment:
Would it be better to define appender with same name and let the following block outside of the if/else ?
```xml
<root level="INFO">
       <appender-ref ref="MYAPPENDER"/>
</root>
```